Client/src/Components/Search/index.jsx

```diff
@@ -26,12 +34,12 @@ export default function Search() {
           />
         )}
         renderTags={(value, getTagProps) =>
-          value.map((option, index) => (
+          value.map((option) => (
             <Chip
-              key={index}
+              key={option.title}
               variant="outlined"
```

Comment:
Not sure if we're ever going to need to use the other variants, but the variant type could potentially be passed in as a prop.  Probably not necessary for now, but something to keep in mind if we need to customize this component further